torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4234,7 +4234,10 @@ def create_symbol(
                 sympy_expr
             ) in config.extended_debug_create_symbol.split(",")
             maybe_more_info = ""
-            if not is_debug:
+            if (
+                not is_debug
+                and os.getenv("TORCH_DISABLE_EXTENDED_DEBUG_INFO") is not None
```

Comment:
nit: can you name this consistently with the other ones? Also, to make it discoverable, I'd tell users about it in the `maybe_more_info` string below.

Reply:
@ezyang thank you for the review! I didn't find a clear pattern to the naming of environment variables, some start with `TORCH`, some with `PYTORCH` (some don't have torch in the name at all), some include the module name some not. Is there a pattern I'm missing?

Anyway, let me know what would be a consistent name and I'll happily adjust and add to the `maybe_more_info` string.

Reply:
The envvar naming is indeed all over the place. But that is no excuse not to at least make them locally consistent. Specifically, the debug messages here refer to `TORCHDYNAMO_EXTENDED_DEBUG_GUARD_ADDED` and similar envvars, so something like `TORCHDYNAMO_EXTENDED_ADVICE=0` feels consistent with what else we have